crates/compression/src/compress.rs

```diff
@@ -104,6 +121,8 @@ where
         transactions,
         #[cfg(feature = "fault-proving")]
         registry_root,
+        #[cfg(feature = "fault-proving")]
+        prev_compressed_block_root,
```

Comment:
We can include the Mekrle root of the `CompressedBlocks` into  `registry_root`, as we do with other roots.

In the proving game we will verify that all entries provided for all tables are correct and connected ot the one `registry_root `.